backend/src/api/events.py

```
@@ -74,6 +81,11 @@ def get_event_service(db: Session = Depends(get_db)) -> EventService:
     return EventService(db=db)
 
 
+def get_conflict_service(db: Session = Depends(get_db)) -> ConflictService:
```

Ruff (0.15.0)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)